### PR TITLE
docs: update injectable migration guide title and schematic description

### PR DESCRIPTION
**Pull request by devversion**
* See individual commits

### CODE REVIEW

**Review by kapunahelewong — APPROVED** [review]
LGTM. Thank you, @devversion!

**Comment by kapunahelewong** [comment]
merge-assistance: Global approval for this docs-only change.

File: aio/content/guide/migration-injectable.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Migration to Add Missing `@Injectable()` Decorators
+# Migration for missing `@Injectable()` decorators and incomplete provider definitions
```
Comment (IgorMinar) [inline aio/content/guide/migration-injectable.md]: this doesn't match the capitalization used in the other guides.

@kapunahelewong is this right? should we change the other guides or fix this one?

**Review by IgorMinar — CHANGES_REQUESTED** [review]
the title looks good, but the capitalization doesn't. Let's see what @kapunahele suggests we do.

File: aio/content/guide/migration-injectable.md (hunk 1 above)
Comment (devversion) [reply aio/content/guide/migration-injectable.md]: Good point. Happy to make it consistent. For what it's worth: looks like some migration guides are not consistent either (e.g. `migration-dynamic-flag`)

Comment (kapunahelewong) [reply aio/content/guide/migration-injectable.md]: Ah, this change is correct (sentence case, that is, only the first word is capitalized and others are lower case unless they are customarily capitalized like `@Injectable()`). I believe when we changed docs teams we went from title case to sentence case. @jbogarthyde, would you like to look into our standard and correct me if I'm missing something here?

Comment (jbogarthyde) [reply aio/content/guide/migration-injectable.md]: That is correct. We have been applying it consistently to internal headings, but not as much to page headings (h1).  
Note that even when the page heading uses sentence case, the entry in the left-nav TOC should still be title case.

**Review by jbogarthyde — APPROVED** [review]

File: aio/content/guide/migration-injectable.md (hunk 1 above)
Comment (kapunahelewong) [reply aio/content/guide/migration-injectable.md]: For reference, @jbogarthyde's PR doing a consistency sweep: https://github.com/angular/angular/pull/34196

Thanks, Judy!

**Pull request closed without merging**